### PR TITLE
Add remove branch

### PR DESCRIPTION
**Pull request by marvin-nyalik**
In this branch the following were done

- Adding new js module with add, remove and updateDescription methods
- Storing the task list item in local storage
- Updating the DOM and task array on addition, removal and update of a task description
- Following the correct gitflow

Let me know of any required changes

### CODE REVIEW

File: src/addRemove.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+};
+
+const updateDesc = (event) => {
+  if (event.key === 'Enter') {
+    const input = event.target;
+    const updatedDescription = input.value.trim();
+    if (updatedDescription !== '') {
+      description.textContent = updatedDescription;
+      updateListItemDescription(item.index, updatedDescription);
+      // toggleTask(item.index); // Show only the to-do-more icon
```
Comment (DammyShittu) [inline src/addRemove.js:62]: - [ ] Kindly remove commented code line. It can make code hard to read.

File: src/index.js [hunk 2]
```diff
@@ -1,27 +1,35 @@
+/* eslint-disable import/no-cycle */
+
 import './style.css';
+import { addTask, removeTask } from './addRemove.js';
+// eslint-disable-next-line import/no-mutable-exports
```
Comment (DammyShittu) [inline src/index.js:5]: - [ ] Linters are often used to maintain code standards. Disabling rules defeats the purpose. Kindly remove these eslint-disables and fix the issues.

File: src/index.js [hunk 3]
```diff
@@ -35,30 +43,85 @@ const handleCheckboxChange = (event) => {
...
+      input.classList.add('edit-description');
+      input.value = description.textContent;
+
+      input.addEventListener('keydown', (event) => {
+        if (event.key === 'Enter') {
+          const updatedDescription = input.value.trim();
+          if (updatedDescription !== '') {
+            description.textContent = updatedDescription;
+            updateListItemDescription(item.index, updatedDescription);
+            // toggleTask(item.index); // Show only the to-do-more icon
```
Comment (DammyShittu) [inline src/index.js:95]: - [ ] Kindly remove commented code.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @marvin-nyalik,

## Highlights
✔️ Add, delete and edit features

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @marvin-nyalik, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**